content/release-10-16-0/applications-10-16-0-bundle/10-16-0-0.md

```diff
@@ -20,6 +20,39 @@ Issue</th>
 </tr>
 </thead><tbody>
 
+<tr>
+<td>Cockpit</td>
+<td>A widget developer can now set the global time context for the whole dashboard or a particular widget. It is also possible to share a dashboard with a specific time context via a URL with the time context parameters applied.</td>
+<td>MTM-48051</td>
+</tr>
+
+<tr>
+<td>Cockpit</td>
+<td>A new generic way of showing alerts in context of specific dashboard widgets has been implemented. For now it is only available in the "Pie chart" widget (users get an alert with a warning about negative values), but the solution is reusable and may be implemented in any widget.</td>

```

Comment:
while not a major issue, i'm not sure about the parentheses here.

Reply:
What exactly do you mean?

Reply:
i'd avoid them and make it a proper subordinate clause